lib/matplotlib/pyplot.py

```diff
@@ -999,9 +999,24 @@ def gcf() -> Figure:
         return figure()
 
 
-def fignum_exists(num: int) -> bool:
-    """Return whether the figure with the given id exists."""
-    return _pylab_helpers.Gcf.has_fignum(num) or num in get_figlabels()
+def fignum_exists(num: int | str) -> bool:
+    """Return whether the figure with the given id exists.
+
+    Parameters
+    ----------
+    num : int or str
+        A figure identifier.
+
+    Returns
+    -------
+    bool
+        Whether or not a figure with id `num` exists.
```

Comment:
```suggestion
        Whether or not a figure with id *num* exists.
```
I agree with @rcomer on the possible reason for failure, but the doc guidelines says that arguments should be within `*` (so it becomes italic, I also agree that monospace is "nicer").

Reply:
Ah yes, you are right of course.